torch_xla/csrc/elementwise.cpp

```diff
@@ -370,22 +370,16 @@ xla::XlaOp BuildLogSigmoidBackward(xla::XlaOp grad_output, xla::XlaOp input,
   return grad_output * (xla::Neg(max_deriv) - sign * (buffer - one) / buffer);
 }
 
-xla::XlaOp BuildElu(xla::XlaOp input, const at::Scalar& alpha,
-                    const at::Scalar& scale, const at::Scalar& input_scale) {
+xla::XlaOp BuildElu(xla::XlaOp input, xla::XlaOp alpha, xla::XlaOp scale,
+                    xla::XlaOp input_scale) {
```

Comment:
you want to be careful about these `at::Scalar` -> `xla::XlaOp` because currently it got default to f64(I think). Maybe we should consider do a `MaybeCast` here.

Reply:
I see, would it be enough to add the `MaybeConvertTo` calls to each of the scalar parameters like below (lines 376 - 379)?

Reply:
I think so, this is equivalent to
```
XlaHelpers::ScalarValue(alpha, shape.element_type(), input.builder())
```

where we upload the scalar to input's type